### DISCUSSION TITLE
resources to learn code and project documentation

### DISCUSSION THREAD

**Discussion by Madhava04A** [EddieHubCommunity/support#7525] [Q&A] [answered]
Hey there, can anyone guide me with steps or resources for technical documentation of a project. from scratch

**Comment by MarcoantonioCaldeira** [comment]
Hello everything is fine? Read this article, I found it very interesting. I'm in Portuguese, but you can translate it:  https://blog.runrun.it/8-documentos-essenciais-para-o-controle-de-projetos/

**Reply by execrc** [reply, marked as answer]
You can refer this resource https://slite.com/learn/technical-documentation
